packages/ERTP/src/config/coreMintKeeper.js

```diff
@@ -1,6 +1,6 @@
 import harden from '@agoric/harden';
 
-import { makeStore } from '../../util/store';
+import makeStore from '@agoric/store';
```

Comment:
`store` is pretty short, is that the right name for this functionality? The downside of splitting these things into new NPM packages is that we're stuck with the names for a while (forever, I guess). Is it `store` as in grocery store, or a place to store data?

Reply:
Store as in store data. Resolving this due to the other discussion thread about Store/WeakStore